src/extension/sdk/dev_tools/manager.ts

```diff
@@ -102,7 +103,16 @@ export class DevToolsManager implements vs.Disposable {
 				return undefined;
 			}
 			this.capabilities.version = installedVersion;
-			if (silent) {
+			if (flutterDaemon != null) {
```

Comment:
nit: if we add `private readonly` in front of `flutterDaemon` in the constructor, that would avoid needing to pass it down through the method calls, and we could check `workspaceContext.config?.startDevToolsFromDaemon` directly here which would be a little more explicit than `flutterDaemon` being set in the case where it's being used for DevTools (otherwise, it's worth a comment noting that's the case).